app/views/projects/show.html.erb

```diff
@@ -59,8 +59,17 @@
         <td class="project-table__cell"></td>
       </tr>
     </tfoot>
-
   </table>
+
+  <% if @sub_projects.any? %>
+    <div class="row">
```

Comment:
I feel this would fit better as a sidebar as navigation (it feels too bulky to have those but links using that much space). Maybe it's better to continue the discussion with nathalie to help with the UX of this PR since she offered.